tests/data/standard/standard_visit.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+__all__ = ["StandardVisit"]
+
+import asyncio
+
+from lsst.ts.scriptqueue.base_script import BaseScript
```

Comment:
Please omit the `.base_script`. BaseScript is available in `lsst.ts.scriptqueue` (that's what the `__init__.py` file is doing in that package).